quasar/main.tf

```diff
@@ -445,7 +445,7 @@ data "aws_ssm_parameter" "prod_password" {
 }
 
 resource "aws_db_instance" "quasar-qa" {
-  allocated_storage               = 1000
+  allocated_storage               = 4000
```

Comment:
How come this was set to a different value than production before?

Reply:
It might also be worth looking into [storage auto-scaling](https://www.terraform.io/docs/providers/aws/r/db_instance.html#storage-autoscaling) in the future.

Reply:
We had smaller data sets in QA vs Prod so didn't justify the additional cost.

Reply:
> It might also be worth looking into [storage auto-scaling](https://www.terraform.io/docs/providers/aws/r/db_instance.html#storage-autoscaling) in the future.

OOOOO sweet, will add a backlog ticket!